.install.sh

```diff
@@ -21,15 +21,21 @@ git checkout "$VERSION"
 {
 	cmake -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON .
 } || {
-	# Install modern CMake
-	mkdir cmake-download
-	pushd cmake-download
-	curl -O https://cmake.org/files/v3.10/cmake-3.10.0-rc5-Linux-x86_64.sh
-	bash cmake-3.10.0-rc5-Linux-x86_64.sh --skip-license
-	export PATH=`pwd`/bin:$PATH
-	echo $PATH
-	popd
-	cmake -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON .
+	machineName=`uname -s`
+	if [ "$machineName" =~ "Linux.*" ]; then
+	  # Install modern CMake
+          mkdir cmake-download
+          pushd cmake-download
+          curl -O https://cmake.org/files/v3.10/cmake-3.10.0-rc5-Linux-x86_64.sh
+          bash cmake-3.10.0-rc5-Linux-x86_64.sh --skip-license
+          export PATH=`pwd`/bin:$PATH
+          echo $PATH
+          popd
+          cmake -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON .
+	else
+	  echo "Failed to find cmake. Please make sure cmake is installed and set in PATH."
```

Comment:
I'd personally prefer it just do this for all operating systems.

Reply:
+1, this code does not appear it would work correctly for x86 Linux, or ARM Linux. My understanding is there is not a technical need for the installation process to pull in a CMake version, so this is something we should leave to the user to supply.

Reply:
I agree. It's also not perfect to modify user's PATH variable. Updated the code change.